MMExtensions/Classes/Extensions/String+Extensions.swift

```diff
@@ -53,9 +53,9 @@ extension String {
         return String(self[index] as Character)
     }
     
-    subscript (range: CountableClosedRange<Int>) -> String {
-        let start = index(startIndex, offsetBy: range.lowerBound)
-        let end = index(startIndex, offsetBy: range.upperBound)
-        return String(self[Range(start ..< end)])
-    }
+//    subscript (range: CountableClosedRange<Int>) -> String {
```

Comment:
@NicolasCombe5555  Please address commented code here!

Reply:
Hello @MuthurajMuthulingam,
That subscript what showing these errors:
.Referencing initializer 'init(_:)' on 'Range' requires that 'String.Index.Stride' conform to 'SignedInteger'
.Referencing initializer 'init(_:)' on 'Range' requires that 'String.Index' conform to 'Strideable'

Reply:
I fixed it for you